packages/@aws-cdk/aws-s3/README.md

```diff
@@ -581,10 +581,7 @@ const bucket = new s3.Bucket(this, 'MyBucket', {
     objectSizeLessThan: 10000, 
     transitions: [{
       storageClass: s3.StorageClass.GLACIER,
-
-      // the properties below are optional
-      transitionAfter: cdk.Duration.days(30),
-      transitionDate: new Date(),
+      transitionMoment: s3.TransitionMoment.after(cdk.Duration.days(30)),
```

Comment:
could we update the readme with a brief section on the `TransitionMoment` explaining what it is?